source/isaaclab/isaaclab/sensors/ray_caster/ray_caster_cfg.py

```diff
@@ -75,7 +85,8 @@ class OffsetCfg:
     """
 
     ray_cast_drift_range: dict[str, tuple[float, float]] = {"x": (0.0, 0.0), "y": (0.0, 0.0), "z": (0.0, 0.0)}
-    """The range of drift (in meters) to add to the projected ray points in local projection frame. Defaults to (0.0, 0.0) for x, y, and z drift.
+    """The range of drift (in meters) to add to the projected ray points in local projection frame. Defaults to
```

Comment:
TODO: @pascal-roth @jsmith-bdai this documentation doesn't clarify at all what this parameter does. Also does it mean we should deprecate "drift_range" at some point? It threw me off why this exists...